src/javascript/gameInstance.js

```diff
@@ -116,69 +130,210 @@ export default class GameInstance {
       "keydown",
       (event) => {
         this.onKeyDown(event.key);
+        if (this.vm.inEditor) {
+          // editor time minor adjust by arrow keys
+          if (event.keyCode === 37) {
```

Comment:
Might wanna pull these key codes into some kind of map?

e.g.

```js
const KeyCode = {
    KEY_LEFT: 37,
};
```

Reply:
Sure, make sense